TechnicalTask-lvl1/UserModule/UserViewController.swift

```diff
@@ -0,0 +1,154 @@
+//
+
+import UIKit
+import Combine
+import CoreData
+
+class UserViewController: UITableViewController {
+    var cancellabels = Set<AnyCancellable>()
+    var fetchedResultsController: NSFetchedResultsController<UserEntityManagedObj>! //persistent
+
+    let keeper = DataKeeper()
+    override func viewDidLoad() {
+        self.tableView.separatorStyle = .none
+        self.tableView.backgroundColor = .white
+        self.tableView.rowHeight = UITableView.automaticDimension
+        self.tableView.estimatedRowHeight = 100
+        self.tableView.register(UINib(nibName: UserCell.xibName, bundle: nil),
+                                forCellReuseIdentifier: UserCell.reuseIdentifier)
+        
+        self.fetchedResultsController = keeper.persistentObservable.fetchedResultsController
+        subscribeOnFRCUpdates()
+        subscribeOnDataUpdate()
+        
+        fetchFromLocal()
+        fetchFromApi()
+    }
+    
+    func subscribeOnFRCUpdates() {
+        keeper.persistentObservable.controllerDidChangeSubject
+            .receive(on: DispatchQueue.main)
+            .sink { changes in
+                if changes.isEmpty {
+                    self.tableView.beginUpdates()
+                    print("->Begin TableView UPDATE")
+                    return
+                }
+                
+                for change in changes {
+                    switch change {
+                    case .inserted(at: let indexPath):
+                        self.tableView.insertRows(at: [indexPath], with: .fade)
+                    case .deleted(from: let indexPath):
+                        self.tableView.deleteRows(at: [indexPath], with: .fade)
+                    }
+                }
+                self.tableView.endUpdates()
+                print("<-End TableView UPDATE")
+            }
+            .store(in: &cancellabels)
+    }
+    
+    func subscribeOnDataUpdate() {
+        keeper.persistentObservable.dataSubject
+            .receive(on: DispatchQueue.main)
+            .sink(receiveCompletion: { completion in
+                if case .failure(let error) = completion {
+                    print("Error fetching users: \(error)")
+                }
+            }, receiveValue: { users in
+                print("Fetched \(users.count) users")
+            })
+            .store(in: &cancellabels)
+    }
+    
+    func fetchFromLocal() {
+        var cancellableFetch: AnyCancellable?
```

Comment:
What is cancellableRequest for here, if there are already cancellables in the viewController?